src/styles/blocks/_catalog.scss

```diff
@@ -0,0 +1,40 @@
+.catalog {
+  display: grid;
+  grid-auto-flow: row;
+  justify-content: center;
+  grid-template-columns: repeat(1, $card-width);
```

Comment:
You don't need to use "repeat" here. You can just set $card-width.

Reply:
Agree